modules/claims_api/spec/requests/v2/veterans/rswag_intent_to_file_request_spec.rb

```diff
@@ -174,8 +174,8 @@
       tags 'Intent to File'
       operationId 'post0966itf'
       security [
-        { productionOauth: ['system/claim.read', 'system/system/claim.write'] },
-        { sandboxOauth: ['system/claim.read', 'system/system/claim.write'] },
+        { productionOauth: ['system/claim.read', 'system/claim.write'] },
+        { sandboxOauth: ['system/claim.read', 'system/claim.write'] },
```

Comment:
found a few duplicate `system/` values merged in to main. this should resolve those. there's a few throughout this PR